docs/sections/user_guide/installation.rst

```diff
@@ -1,5 +1,67 @@
+.. include:: ../links.rst
+
 ************
 Installation
 ************
 
-TBD
+
+-  If you are a Developer, skip the User Installation and continue on to the Developer Setup located in the Contributer Guide.
```

Comment:
Sorry to force you to be the pioneer again, but... Is there an RST way to make `Developer Setup` and `Contributor Guide` hyperlinks to those places in the docs? It'd be nice if we could.

Also, spelling, `Contributer` -> `Contributor`.

Reply:
I think it would help, if it's possible, to use relative links within our docs so that they always refer to the same build of the docs. I think that if we use absolute https:// URLs like these, a page in, say, a PR build of the docs will link to the official docs rather than to another page in the same PR build.

Reply:
All comments should be addressed now besides this one. I'm having a hard time interrupting what you mean here. But I did a bit of research and I think maybe this is what you'd like to see:

https://stackoverflow.com/questions/3630386/insert-relative-link-in-restructuredtext
https://www.odoo.com/documentation/17.0/contributing/documentation/rst_guidelines.html

Trying to implement either of these suggestions so far hasn't worked for me. Still trying to make progress on this.

Reply:
If you're happy with this solution Paul, I'll merge it. Just want to be sure I covered this.

Reply:
Sorry for the delay -- everything looks good to me, thank you!